conf/zoo_sample.cfg

```diff
@@ -6,6 +6,10 @@ initLimit=10
 # The number of ticks that can pass between 
 # sending a request and getting an acknowledgement
 syncLimit=5
+# enable TTL Nodes
+# IMPORTANT: when enabled, your server ID cannot be greater than 254
+# See https://zookeeper.apache.org/doc/r3.5.3-beta/zookeeperAdmin.html#sc_configuration for details
+ttlNodesEnabled=false
```

Comment:
I was actually thinking that this specific line (not in general) should be true by default. That way anyone starting out would get the feature enabled by default - and they wouldn't have any legacy serverids specified. What do you think?